gen2-gaze-estimation/main.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+from os import pipe
```

Comment:
any reason for this?

Reply:
I'm not sure, since I'm not the one who put that there. Removed it, since it's unnecessary.

Reply:
Hmm... I might have hit some key combination to import that, or VSCode did it automatically, not sure. Either way it's irrelevant.